### ISSUE TITLE
charla avanzada sobre git

### ISSUE THREAD

**Issue by rafaelleru** [iblancasa/miercolesgeek#45]
El zen de git esta bien pero estaria bien algo mas avanzado, en donde se comente como gestionar los conflictos, trabajar en repos con muchos forks adecuadamente, etc.

**Comment by angrykoala** [comment]
El Zen de Git 2: El retorno del Jedi

**Comment by 29antonioac** [comment]
Yo lo veo!

**Comment by JJ** [comment]
¿para cuando? ¿Mañana o tarde?

**Comment by rafaelleru** [comment]
Segun comentaste el otro dia los jueves hay un alula reservada, asi que un jueves por la mañana podria estar bien.

**Comment by rafaelleru** [comment]
@JJ

**Comment by JJ** [comment]
Sí, lo que ocurre es que tenemos reservada sólo media hora, y no sé yo si
para una avanzada daría tiempo... Podríamos proponerlo para algún miércoles
o martes.

**Comment by rafaelleru** [comment]
pues cualquier miercoles que se pueda!!!
